### PR TITLE
move from beta to preview

### PR DESCRIPTION
**Pull request by acosferreira**
# Description

Change /beta to /preview following [RHCLOUD-24353](https://issues.redhat.com/browse/RHCLOUD-24353)

**Should be merged May 1st**

Fixes # (THEEDGE3129)

## Type of change

What is it?

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Documentation update
- [ ] Tests update

# Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] I run `npm run lint:js:fix` to check that my code is properly formatted

### CODE REVIEW

**Comment by acosferreira** [comment]
@matoval @resoluteCoder ^^^

**Review by matoval — APPROVED** [review]

File: profiles/local-frontend.js [hunk 1]
```diff
@@ -5,13 +5,13 @@ const APP_ID = 'edge';
...
 
-routes[`/beta/${SECTION ? `/${section}` : ''}${APP_ID}`] = {
+routes[`/preview/${SECTION ? `/${section}` : ''}${APP_ID}`] = {
   host: `http://localhost:${FRONTEND_PORT}`,
 };
 routes[`/${SECTION ? `/${section}` : ''}${APP_ID}`] = {
   host: `http://localhost:${FRONTEND_PORT}`,
 };
-routes[`/beta/apps/${APP_ID}`] = { host: `http://localhost:${FRONTEND_PORT}` };
+routes[`/preview/apps/${APP_ID}`] = { host: `http://localhost:${FRONTEND_PORT}` };
```
Comment (Hyperkid123) [inline profiles/local-frontend.js:14]: Routing does not change for asset deployment. Only for browser routes.

File: config/dev.webpack.config.js [hunk 2]
```diff
@@ -21,7 +21,7 @@ const webpackProxy = {
       '/api/edge': { host: `http://localhost:${process.env.API_PORT}` },
     }),
     ...(process.env.CONFIG_PORT && {
-      [`${process.env.BETA ? '/beta' : ''}/config`]: {
+      [`${process.env.BETA ? '/preview' : ''}/config`]: {
```
Comment (Hyperkid123) [inline config/dev.webpack.config.js:24]: This route does not change.

File: config/dev.webpack.config.js [hunk 3]
```diff
@@ -3,16 +3,16 @@ const config = require('@redhat-cloud-services/frontend-components-config');
...
+  ...(process.env.BETA && { deployment: 'preview/apps' }),
 };
 
 const webpackProxy = {
-  deployment: process.env.BETA ? 'beta/apps' : 'apps',
+  deployment: process.env.BETA ? 'preview/apps' : 'apps',
   useProxy: true,
   env: `${process.env.ENVIRONMENT || 'stage'}-${
-    process.env.BETA ? 'beta' : 'stable'
+    process.env.BETA ? 'preview' : 'stable'
```
Comment (Hyperkid123) [inline config/dev.webpack.config.js:13]: This does not change to preview

File: config/dev.webpack.config.js [hunk 4]
```diff
@@ -3,16 +3,16 @@ const config = require('@redhat-cloud-services/frontend-components-config');
 
 const insightsProxy = {
   https: false,
-  ...(process.env.BETA && { deployment: 'beta/apps' }),
+  ...(process.env.BETA && { deployment: 'preview/apps' }),
 };
 
 const webpackProxy = {
-  deployment: process.env.BETA ? 'beta/apps' : 'apps',
+  deployment: process.env.BETA ? 'preview/apps' : 'apps',
```
Comment (Hyperkid123) [inline config/dev.webpack.config.js:10]: Deployment can't change. Only browser routing does.

File: config/dev.webpack.config.js [hunk 5]
```diff
@@ -3,16 +3,16 @@ const config = require('@redhat-cloud-services/frontend-components-config');
 
 const insightsProxy = {
   https: false,
-  ...(process.env.BETA && { deployment: 'beta/apps' }),
+  ...(process.env.BETA && { deployment: 'preview/apps' }),
```
Comment (Hyperkid123) [inline config/dev.webpack.config.js:6]: Deployment can't change. Only browser routing does.

**Pull request closed without merging**